src/app/components/ATIAnalytics/params/index.ts

```diff
@@ -59,7 +59,7 @@ const ARTICLE_PHOTO_GALLERY = 'article-photo-gallery';
 const ARTICLE_CORRESPONDENT_PIECE = 'article-correspondent';
 const ARTICLE_SHORT_FORM_VIDEO = 'article-sfv';
 
-const MIGRATED_PAGE_TYPES: PageTypes[] = [HOME_PAGE, ARTICLE_PAGE];
+const MIGRATED_PAGE_TYPES: PageTypes[] = [HOME_PAGE, ARTICLE_PAGE, TOPIC_PAGE];
```

Comment:
Should `[TOPIC_PAGE]` now be a `no0p` on line 94?

Reply:
I think so yes - I did this for the MostRead page and it didn't cause any problems -> https://github.com/bbc/simorgh/pull/10980/

Could we also delete the `src/app/components/ATIAnalytics/params/topicPage` directory now?

Reply:
made it `noOp`